src/services/validators/createServiceDto.validator.ts

```diff
@@ -0,0 +1,28 @@
+import { IValidator, ValidatedResponse } from '../../shared/validators/validtor.interface';
+import { CreateServiceRequestDto } from '../models/dtos/service/request/create-service.requrest.dto';
+
+export class CreateServiceDtoValidator implements IValidator<CreateServiceRequestDto> {
+    validate(obj: CreateServiceRequestDto): ValidatedResponse {
+        const errorMessages = [];
+        if (obj.name == null || obj.name.trim().length <= 0) {
```

Comment:
don't use equality comparison of NULL and prefer '===' over '=='

Reply:
applied